pytest_html/extras.py

```diff
@@ -8,17 +8,33 @@
 FORMAT_TEXT = 'text'
 FORMAT_URL = 'url'
 
+TYPE_JPG = ('image/jpeg', 'jpg')
+TYPE_PNG = ('image/png', 'png')
+TYPE_SVG = ('image/svg+xml', 'svg')
 
-def extra(content, format, name=None):
-    return {'name': name, 'format': format, 'content': content}
+
+def extra(content, format, name=None, type=None):
+    return {'name': name, 'format': format, 'content': content, 'type': type}
 
 
 def html(content):
     return extra(content, FORMAT_HTML)
 
 
 def image(content, name='Image'):
```

Comment:
We can expose `mime_type` and `extension` as optional arguments, defaulting to the values for PNG.